log2.log

```diff
@@ -0,0 +1,336 @@
+DEBUG rustc_resolve resolve_crate_root($crate#0)
+DEBUG rustc_resolve resolve_crate_root: marks=[]
+DEBUG rustc_resolve resolve_crate_root: found opaque mark None None
+DEBUG rustc_resolve resolve_crate_root: found semi-transparent mark None None
+DEBUG rustc_resolve resolve_crate_root($crate#0): found no mark (ident.span = no-location (#0))
+DEBUG rustc_resolve resolve_crate_root($crate#1)
+DEBUG rustc_resolve resolve_crate_root: marks=[(ExpnData { kind: AstPass(StdImports), parent: crate0::{{expn0}}, call_site: no-location (#0), disambiguator: 0, def_site: no-location (#0), allow_internal_unstable: Some(["prelude_import"]), edition: Edition2015, macro_def_id: None, parent_module: None, allow_internal_unsafe: false, local_inner_macros: false, collapse_debuginfo: false }, Opaque)]
```

Comment:
looks like the file was committed on accident? :sweat_smile:

Reply:
🤣，sorry for that